src/Bicep.LangServer/Completions/CompletionItemBuilder.cs

```diff
@@ -10,9 +10,37 @@ public static class CompletionItemBuilder
     {
         public static CompletionItem Create(CompletionItemKind kind) => new CompletionItem {Kind = kind};
 
-        public static CompletionItem WithLabel(this CompletionItem item, string label)
+        public static CompletionItem WithAdditionalEdits(this CompletionItem item, TextEditContainer editContainer)
```

Comment:
>WithAdditionalEdits [](start = 37, length = 19)

Are there any changes here? Or just reorders?

Reply:
Just alphabetical reorders

Reply:
You'll likely have conflicts with my PR because I modified this file as well.

---
In reply to: [551621280](https://github.com/Azure/bicep/pull/1236#discussion_r551621280) [](ancestors = 551621280)